functions.go

```diff
@@ -272,4 +272,7 @@ var genericMap = map[string]interface{}{
 	"regexReplaceAll":        regexReplaceAll,
 	"regexReplaceAllLiteral": regexReplaceAllLiteral,
 	"regexSplit":             regexSplit,
+
+	// Localization
+	"t": locale.T,
```

Comment:
In this case locale is a package level variable. When performing actions concurrently there is a chance for a race condition when concurrence is handling multiple languages. For example,

* concurrent request 1 sets the language to FR
* concurrent request 2 sets the language to EN
* concurrent request 1 calls ExectuteTeample to generate output. Expects FR but gets EN
* concurrent request 2 calls ExectuteTeample to generate output. Expects EN but gets EN

This might work for a single user environment but won't work when we get into uses for multiple users.

Reply:
agree, will take a look how i can change it, so in such case, we have to decline the idea to have predefined global language, so on every call, the translation should be set seperatly.